src/osemgrep/reporting/Matches_report.ml

```diff
@@ -277,30 +286,63 @@ let pp_text_outputs ~max_chars_per_line ~max_lines_per_finding ~color_output ppf
       | Some m -> m <> cur.extra.message
     in
     if print then (
-      (* The 24m is "no underline", and for python compatibility *)
-      let esc =
-        if Fmt.style_renderer ppf = `Ansi_tty then Fmt.any "\027[24m"
-        else Fmt.any ""
+      let sev_icon_enablement = false in
+      (* Enable in a separate PR post-test updates *)
+      let pp_styled_severity =
```

Comment:
so this code is never executed?

And osempass is still fine because it's gaurded by the new cli_ux in pysemgrep?
(and our snapshot tests are using the old_cli_ux?)

Reply:
yes that's correct! I'll make a note of it here.